src/check.py

```diff
@@ -403,14 +403,16 @@ def check_answer_contains_points(doc:Document,start:int,end:int):
         
         if "記述設問" in question:
             if "解答のポイント" not in question:
-                # 文字を丸める
                 error_question = ""
-                if len(question) > 15:
-                    error_question = question[:15] + "…"
-                else:
-                    error_question = question
+                # 漢数字を取得しアラビア文字に変換
+                if kanji_question_pattern.match(question):
+                    kanji_number = kanji_question_pattern.match(question).group(1)
+                    error_question = src.doc_util.kanji_number_to_arabic_number(kanji_number)
```

Comment:
```suggestion
                matches = kanji_question_pattern.match(question)
                if matches:
                    error_question = src.doc_util.kanji_number_to_arabic_number( matches.group(1))
```

kanji_question_pattern.match(question)が2回評価されていて、無駄かなと思いました

Reply:
ここマッチしないとどうなるのでしょうか？
そもそもマッチしないことあるのでしょうか？

src.doc_util.get_explanation_of_questionsが"問"を手がかりに、
リストに詰めているので、必ずマッチするように思えます。
であるなら、
question_explanation_listをenumelateで回してindexをとったほうが良さそうです。